code/modules/modular_computers/file_system/programs/arcade.dm

```diff
@@ -27,6 +27,7 @@
 
 /datum/computer_file/program/arcade/proc/game_check(mob/user)
 	sleep(5)
+	user?.mind.adjust_experience(/datum/skill/gaming, 1)
```

Comment:
If user gets a ``?.`` mind should also get a ``.?``.
Think of all the people that ghost while playing vidya.

Reply:
is ? here an if check

Reply:
``user?.mind.adjust_experience(/datum/skill/gaming, 1)``
translates into
``user && user.mind.adjust_experience(/datum/skill/gaming, 1)``

Reply:
Is this serious? I think if the user reaches this point they should necessarily have a mind

Reply:
the mob could potentially get gibbed during the sleep

not really a massive concern, but a sanity check is pretty much free